### ISSUE TITLE
(Enhancement) DELCS: Implement a depth-based fade out.

### ISSUE THREAD

**Issue by Runbiorn** [martymcmodding/qUINT#22]
I think implementing something like MXAO's depth falloff would greatly improve the shader.

DELCS has become my favorite sharpening method and this is the only feature I miss from other implementations.

Thank you for sharing your work! 🙂

**Comment by martymcmodding** [comment]
Good point. I wanted to update DELC Sharpen based on insight gained with AMD FSR anyways, might as well add this as well. Will do. Can't say when because my schedule is a bit tight at the moment, but I'll add it.
